app/logic/Mail/ActiveSync/ActiveSyncAccount.ts

```diff
@@ -189,17 +189,17 @@ export class ActiveSyncAccount extends MailAccount {
     if (response.status == 401) {
       const repeat = async () => {
         this.retries++;
-        let result = await this.verifyLogin(); // repeat the call
+        await this.verifyLogin(); // repeat the call
         this.retries = 0;
-        return result;
```

Comment:
Still removing the return?

Reply:
There is no else block here; you're looking at the wrong place.

Reply:
I see that you remove the "return;" on line 194, there is an "if" on line 196, and an "else if" on lines 199 and 203, another "else" on line 206, and a "throw" on line 211.
At least that's what GitHub shows me here.
If you remove the "return", you fall into one of these throws.

Unless you are the very end of the function, could you please just keep the "return" in there?

Reply:
Oh, this is an inline function created with const =. I couldn't see that correctly here on the small screen.